app/models/lsp/LSPSummary.scala

```diff
@@ -18,11 +18,14 @@ package models.lsp
 
 import play.api.libs.json.{Format, Json}
 
+import java.time.LocalDate
+
 case class LSPSummary(
                        activePenaltyPoints: Int,
                        inactivePenaltyPoints: Int,
                        regimeThreshold: Int,
-                       penaltyChargeAmount: BigDecimal
+                       penaltyChargeAmount: BigDecimal,
+                       PoCAchievementDate: LocalDate
                      )
```

Comment:
Do we not want to add 'PoCAchievementDate' to LSPSummary model in backend

Reply:
Already done here: https://github.com/hmrc/penalties/pull/169

Reply:
> Already done here: [hmrc/penalties#169](https://github.com/hmrc/penalties/pull/169)

Ah! sorry I missed it.

Reply:
Runs ok against the AT branch PRM-1594